pom.xml

```diff
@@ -40,6 +40,8 @@
 		<initializer.version>2.3.0</initializer.version>
 		<appointments.version>1.4.0</appointments.version>
 		<fhir2.version>1.7.0</fhir2.version>
+		<webservices.restVersion>2.33.0</webservices.restVersion>
```

Comment:
This also needs to be declared in config.xml

Reply:
Done